src/components/text/Text.props.ts

```diff
@@ -0,0 +1,22 @@
+import { PType } from '../../foundations/type';
+
+import { TTextSize, TTextElement, TTextWeight } from './Text.types';
+
+export type PText = Pick<PType, 'color' | 'margin'> & {
+    className?: string;
+    /**
+     * the size-token used to render the text size.
+     * @default '100'
+     * */
+    size?: TTextSize;
+    /**
+     * for text components the options are `p` and `span`
+     * @default 'p'
+     * */
+    element?: TTextElement;
+    /**
+     * define the weight of the rendered text
+     * @default 'regular'
+     * */
+    weight?: TTextWeight;
```

Comment:
The `Heading` component includes this using the `Pick` TS operator, should we do the same here? Or maybe mirror this on the `Heading` component instead?

Just wondering again about consistency between the two convenience components.

Reply:
I changed the way it is being used in both `Heading` and `Text` components.